### DISCUSSION TITLE
export source code

### DISCUSSION THREAD

**Discussion by machinefriendly** [ToolJet/ToolJet#5998] [General]
Hi, I see that we could export .json file for app configuration, which will be used inside Tooljet to recreate an app in other workspace, I'm wondering if there is possibility to export the entire application's source code so that we could deploy app as we want or for a further modification which cannot be done by Tooljet yet.

Thanks.

**Comment by withshubh** [comment]
> Hi, I see that we could export .json file for app configuration, which will be used inside Tooljet to recreate an app in other workspace, I'm wondering if there is possibility to export the entire application's source code so that we could deploy app as we want or for a further modification which cannot be done by Tooljet yet.
> 
> Thanks.

Hey @machinefriendly - ToolJet applications can only be used within the ToolJet platform. You cannot deploy the ToolJet apps separately. Could you please share more about your use-case?

**Reply by machinefriendly** [reply]
Thanks for your feedback. That's not what we expect, it looks like that Tooljet is a open source version of Bubble, the app generated from which platform cannot be run outside, we cannot use Tooljet to quickly generate an app and modify its front/back end codes if necessary by developers and then deploy it as any other applications.

**Reply by withshubh** [reply]
@machinefriendly - We don't support code export and it isn't in our roadmap.
